cross/python38/Makefile

```diff
@@ -0,0 +1,109 @@
+PKG_NAME = python38
+PKG_VERS = 3.8.6
```

Comment:
as of December 21 2020 the release 3.8.7 is officially available, so I propose to update to the current 3.8 version

Reply:
Done.